pdc/settings_local.py.dist

```diff
@@ -24,6 +24,8 @@
 # checks, uncomment this line, set DISABLE_RESOURCE_PERMISSION_CHECK with True.
 #     DISABLE_RESOURCE_PERMISSION_CHECK = True
 #
+# Example 3: set the id of sigkey which has name "redhatrelease2" as default
```

Comment:
From this description I still wouldn't know what the option does.

Reply:
Add more comment